ibis/expr/types/relations.py

```diff
@@ -92,6 +92,9 @@ class Table(Expr, JupyterMixin):
     def __array__(self, dtype=None):
         return self.execute().__array__(dtype)
 
+    def as_table(self):
```

Comment:
Shall we add `as_table` to the base Expr as an abstractmethod?

Reply:
Hm, I thought I did. I'll do that if not.

Reply:
Done!